lumicks/pylake/channel.py

```diff
@@ -21,14 +22,54 @@ class Slice:
         Plot labels: "x", "y", "title".
     calibration: ForceCalibration
     """
-    def __init__(self, data_source, labels=None, calibration=None):
+    def __init__(self, data_source, labels=None, calibration=None, unit=None):
```

Comment:
If `unit == None` the same thing as `unit == "dimensionless"`? Should it be?

Reply:
No unit (None) would be the true arbitrary unit I suppose, i.e. no information on the unit being available. Dimensionless was renamed to make this more clear.